handbook/engineering/Debugging.md

```diff
@@ -134,7 +134,7 @@ allow us to see more details as to what's happening in the communication with Fl
 the logs and other configurations.
 
 If you are running Fleetd, you should add `--debug` to the command-line options. This will get debug logs for Fleetd and 
-also for osquery automatically. Check the [Fleetd Docs](https://fleetdm.com/docs/using-fleet/fleetd#logs) 
+also for osquery automatically. Check the [fleetd docs](../../docs/Using%20Fleet/enroll-hosts.md#finding-fleetd-logs) 
```

Comment:
handbook changes lgtm, though
- (a) I'd consider sticking to full canonical URLs instead of relative paths so that everything still works if this text gets pasted somewhere else (even though I realize this means that it is potentially less convenient to navigate on github.com for the rare users doing that instead of using the website)
- (b) I'm pretty sure that relative links between handbook and docs don't actually work properly (I remember writing a comment about this at some point- it's fixable just hasn't been worth spending time on making it work, esp. since using canonical fleetdm.com URLs tends to be more resillient anyways)

Reply:
Updated the PR to use a URL instead of relative path